pkg/controller/credentialsrequest/credentialsrequest_controller_test.go

```diff
@@ -684,6 +685,24 @@ func TestCredentialsRequestReconcile(t *testing.T) {
 				assert.NoError(t, err)
 			},
 		},
+		{
+			name: "skip sync if last sync less than 1 hour ago (and lastsyncgeneration is current)",
```

Comment:
This'll look a little extreme if you go to run a single test on CLI, maybe trim down to something close to: "skip AWS if recently synced"

We'd want a test for "sync when recently synced but modified" as well.